kuma/static/styles/components/banners/browser-compat-report.scss

```diff
@@ -0,0 +1,170 @@
+@import '../../includes/vars';
+@import '../../includes/mixins';
+@import 'vars';
+
+.developer-needs {
```

Comment:
Obviously, this is related to that `classname` being a bit "wrong". I wouldn't mind you taking the time to come up with a generic name instead so this block of scss can be shared. Or, let this new banner use `developer-needs.scss` but I noticed that the SCSS isn't actually identical.